app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -13,84 +12,150 @@ import androidx.compose.foundation.lazy.items
+import kotlinx.coroutines.launch
 
 @Composable
 fun PartiesScreen(
 	viewModel: PartiesViewModel,
 	navController: NavHostController,
 ) {
 	val state by viewModel.state
+
 	when {
 		state.isLoading() -> PicoverLoader(modifier = Modifier.fillMaxSize())
 		state.isLoaded() -> LoadedContent(
 			parties = state.data(),
-			openDetails = {
+			onPartyClicked = {
 				navController.navigate("partyDetails/$it")
 			},
+			onTitleChange = { viewModel.updateTitle(it) },
+			titleValidationStatus = viewModel.validateShortText(viewModel.title),
+			title = viewModel.title,
+			description = viewModel.description,
+			onDescriptionChange = { viewModel.updateDescription(it) },
+			descriptionValidationStatus = viewModel.validateLongText(viewModel.description),
+			cleanBottomSheetFields = { isBottomSheetClosed ->
+				if (isBottomSheetClosed) {
+					viewModel.updateTitle("")
+					viewModel.updateDescription("")
+				}
+			},
 		)
 
 		state.isError() -> PicoverGenericError(onRetryClick = { viewModel.loadParties() })
 	}
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-private fun LoadedContent(parties: List<Party>, openDetails: (Int) -> Unit) {
-	val context = LocalContext.current
-	Box(Modifier.fillMaxSize()) {
+private fun LoadedContent(
+	parties: List<Party>,
+	onPartyClicked: (Int) -> Unit,
+	cleanBottomSheetFields: (Boolean) -> Unit,
+	title: String,
+	onTitleChange: (String) -> Unit,
+	titleValidationStatus: ValidationStatus,
+	description: String,
+	onDescriptionChange: (String) -> Unit,
+	descriptionValidationStatus: ValidationStatus,
+) {
+	val sheetState = rememberModalBottomSheetState()
+	val coroutineScope = rememberCoroutineScope()
+
+	Box(
+		modifier = Modifier
+			.fillMaxSize()
+			.padding(16.dp),
+	) {
 		LazyColumn(
-			modifier = Modifier
-				.fillMaxSize()
-				.padding(16.dp),
+			modifier = Modifier.padding(16.dp),
 			verticalArrangement = Arrangement.spacedBy(16.dp),
 		) {
 			items(parties) {
 				PartyTile(
 					party = it,
-					openDetails = openDetails,
+					onPartyClicked = onPartyClicked,
 				)
 			}
 		}
 		FloatingActionButton(
 			modifier = Modifier
-				.align(Alignment.BottomEnd)
-				.padding(12.dp),
-			onClick = { Toast.makeText(context, "To be implemented", Toast.LENGTH_SHORT).show() },
+				.padding(12.dp)
+				.align(Alignment.BottomEnd),
+			onClick = {
+				coroutineScope.launch {
+					sheetState.show()
+				}
+			},
 			containerColor = MaterialTheme.colorScheme.secondaryContainer,
 			contentColor = MaterialTheme.colorScheme.secondary,
 		) {
-			Icon(Icons.Filled.Add, "null")
+			Icon(Icons.Filled.Add, stringResource(id = R.string.FabAddIcon))
+		}
+		if (sheetState.isVisible) {
+			ModalBottomSheet(
+				modifier = Modifier
+					.padding(bottom = 56.dp),
+				sheetState = sheetState,
+				onDismissRequest = {
+					coroutineScope.launch {
+						sheetState.hide()
+					}
+				},
+			) {
+				PartyBottomSheet(
+					nameValue = title,
```

Comment:
name and title, please use one naming convention

Reply:
Changed to the title in all places. Thank you